pkg/test/resourcebuilder/control_plane_machine_set.go

```diff
@@ -23,17 +23,29 @@ import (
 )
 
 const (
-	// controlPlaneMachineSetName is the only valid name allowed.
+	// ControlPlaneMachineSetName is the only valid name allowed.
 	// A ControlPlaneMachineSet is a singleton within the cluster, this matches other singletons such as Infrastructure.
-	controlPlaneMachineSetName = "cluster"
+	ControlPlaneMachineSetName = "cluster"
 )
 
+type CPMS interface {
+	Build() *machinev1.ControlPlaneMachineSet
+}
+
+type NakedControlPlaneMachineSet struct {
+	Cpms *machinev1.ControlPlaneMachineSet
+}
```

Comment:
Lets make this `Raw` like this, again, common naming we see in other places in kube
```suggestion
type RawControlPlaneMachineSet struct {
	Raw *machinev1.ControlPlaneMachineSet
}
```